docs/en/registry.rst

```diff
@@ -559,23 +610,23 @@ Digital Credentials Catalog contents is secured in a JWS that contains the follo
 
 The JWS payload contains the following parameters:
 
-.. list-table:: First-level Fields of the Catalog
+.. list-table:: First-level Fields of the Digital Credentials Catalog
    :class: longtable
    :header-rows: 1
    :widths: 30 70
 
    * - Field Name
      - Description
-   * - **catalog_version**
+   * - **version**
      - REQUIRED. Version of the Digital Credential Catalog format.
-   * - **iss**
-     - REQUIRED. Issuer identifier of the Digital Credential Catalog.
    * - **last_modified**
      - REQUIRED. Timestamp of the last modification to the Digital Credential Catalog.
+   * - **iss**
+     - REQUIRED. Issuer identifier of the Digital Credential Catalog.
    * - **credentials**
      - REQUIRED. Array containing Digital Credential definitions.
-   * - **wallet_app_attestation**
-     - REQUIRED. A JSON Object containing definitions for Wallet App Attestations, including their supported formats, and associated claims. This Object is used by other entities, such as Issuers and Relying Parties, to retrieve information about the Wallet App Attestation formats supported within the ecosystem.
+   * - **wallet_app_attestations**
+     - REQUIRED. A JSON Array containing definitions for Wallet App Attestations, including their supported formats, and associated claims. This Object is used by other entities, such as Issuers and Relying Parties, to retrieve information about the Wallet App Attestation formats supported within the ecosystem.
```

Comment:
we should say about the format of each array entry

Reply:
please provide an example of a single entry, I would like to check it.


Generally, it would be better to keep it as key:value instead of array